src/index.tsx

```diff
@@ -8,27 +8,13 @@ import {
   gql,
 } from '@apollo/client';
 import './index.css';
+import App from './App';
 
 const client = new ApolloClient({
   uri: 'http://54.147.153.168',
```

Comment:
Since this endpoint does not, I see no reason to keep this lines of code that reference graphql

Reply:
We are going to need this later